docs/developer/sdk_from_swagger_and_typespec.md

```diff
@@ -0,0 +1,195 @@
+# Overview
+
+This document compares Python SDKs generated from Swagger (OpenAPI) specifications versus TypeSpec. For clarity, we'll refer to these as "Swagger SDKs" and "TypeSpec SDKs" respectively.
+
+## Model 
+
+### Model Structure
+
+#### Msrest Model
+
+Swagger SDKs are generated from [Swagger specifications](https://github.com/Azure/azure-rest-api-specs/tree/main/specification) using [@autorest/python](https://www.npmjs.com/package/@autorest/python), and implement the Msrest model pattern. The following example illustrates the fundamental structure of an Msrest model:
+
+```python
+from azure.mgmt.example._utils import serialization as _serialization
+
+class Person(_serialization.Model):
+    _attribute_map = {
+        "name": {"key": "name", "type": "str"},
+        "parent_name": {"key": "parentName", "type": "str"},
+    }
+
+    def __init__(self, *, name: Optional[str] = None, parent_name: Optional[str] = None) -> None:
+        ...
+```
+
+### DPG Model
+
+TypeSpec SDKs are generated from [TypeSpec](https://github.com/microsoft/typespec/) using [@azure-tools/typespec-python](https://www.npmjs.com/package/@azure-tools/typespec-python), and implement the DPG model pattern. The following example demonstrates the fundamental structure of a DPG model:
+
+```python
+from azure.mgmt.example._utils.model_base import Model as _Model, rest_field
```

Comment:
i still think we should get rid of the imports since it's kind of ugly, has a lot of private stuff in it. Better to just focus on what the generated model will look like, and how to use it

Reply:
Updated.